src/middleware/assets.js

```diff
@@ -77,6 +80,20 @@ Or \`rm -rf bower_components/n-ui && bower install n-ui\` if you're no longer wo
 			return currentHeadCsses;
 		}, {}) : {};
 
+	if (process.NODE_ENV !== 'production') {
+		const paths = Object.keys(headCsses).map(css => `${directory}/public/${css}.css`);
+		chokidar.watch(paths).on('change', (path) => {
+			fs.readFile(path, 'utf-8', (err, content) => {
```

Comment:
could use Matt's great https://github.com/matthew-andrews/denodeify lib to turn this into a promise